src/api/app/helpers/webui/project_helper.rb

```diff
@@ -3,6 +3,16 @@ module Webui::ProjectHelper
 
   protected
 
+  def pulse_period(range)
+    start = if range == 'month'
+              Time.zone.today.prev_month.strftime('%B, %e')
+            else
+              Time.zone.today.prev_week.strftime('%B, %e')
```

Comment:
[nitpick] I would say we can avoid having  repeated code in those two lines :thinking: